vizro-core/tests/unit/vizro/models/_components/form/test_range_slider.py

```diff
@@ -0,0 +1,331 @@
+                    html.Div(
+                        [
+                            dcc.Input(
+                                id="range_slider_with_all_start_value",
+                                type="number",
+                                placeholder="start",
+                                min=0,
+                                max=10,
+                                className="slider_input_field_left",
+                                value=0,
+                                size="24px",
+                                persistence=True,
+                            ),
+                            dcc.Input(
+                                id="range_slider_with_all_end_value",
+                                type="number",
+                                placeholder="end",
+                                min=0,
+                                max=10,
+                                className="slider_input_field_right",
+                                value=10,
+                                persistence=True,
+                            ),
+                            dcc.Store(id="temp-store-range_slider-range_slider_with_all", storage_type="local"),
+                        ],
+                        className="slider_input_container",
+                    ),
+                ],
+                className="range_slider_inner_container",
+            ),
+        ],
+        className="selector_container",
+        id="range_slider_with_all_outer"
+    )
+
+
+class TestRangeSliderInstantiation:
+    """Tests model instantiation."""
+
+    def test_create_range_slider_mandatory_only(self):
+        range_slider = vm.RangeSlider()
+
+        assert hasattr(range_slider, "id")
+        assert range_slider.type == "range_slider"
+        assert range_slider.min is None
+        assert range_slider.max is None
+        assert range_slider.step is None
+        assert range_slider.marks is None
+        assert range_slider.value is None
+        assert range_slider.title is None
+        assert range_slider.actions == []
+
+    def test_create_range_slider_mandatory_and_optional(self):
+        range_slider = vm.RangeSlider(min=0, max=10, step=1, marks={}, value=[1, 9], title="Test title")
+
+        assert range_slider.min == 0
+        assert range_slider.max == 10
+        assert range_slider.step == 1
+        assert range_slider.value == [1, 9]
+        assert range_slider.title == "Test title"
+        assert range_slider.actions == []
+
+    @pytest.mark.parametrize(
+        "min, max",
+        [
+            (0, None),
+            (None, 10),
+            (0, 10),
+        ],
+    )
+    def test_valid_min_max(self, min, max):
+        range_slider = vm.RangeSlider(min=min, max=max)
+        min = int(min) if isinstance(min, str) else min
+        max = int(max) if isinstance(max, str) else max
+
+        assert range_slider.min == min
+        assert range_slider.max == max
+
+    def test_invalid_min_max(self):
```

Comment:
As an overall guidance, ideally also changed for the slider tests, could we name the tests:

`test_validate_max_xyz`, where `xyz` could indeed be `invalid` or so. This would make it easier to see which validator you are even trying to trigger